website/client/components/ui/drawerSlider.vue

```diff
@@ -115,45 +117,58 @@
           next,
         }),
         currentWidth: 0,
-        currentPage: 0,
+        pointer: 0,
       };
     },
     computed: {
-      pages () {
-        return _chunk(this.items, this.itemsPerPage() - 1).map((content, index, array) => {
-          let resultData = [...content];
-
-          if (array[index + 1]) {
-            resultData.push({
-              ...array[index + 1][0],
-              ofNextPage: true,
-            });
-          }
-
-          return resultData;
-        });
+      showItems () {
+        let items = this.items;
+        let pointer = this.pointer;
+        let itemsPerPage = this.itemsPerPage();
+        let firstSlice = items.slice(pointer, pointer + itemsPerPage);
+
+        if (firstSlice.length === itemsPerPage) {
+          return firstSlice;
+        } else {
+          let getRemainderItems = Math.abs(itemsPerPage - firstSlice.length);
```

Comment:
`itemsPerPage` and `firstSlice.length` shouldn't be less than 0, so I guess there is no need for `Math.abs`